packages/nc-gui/utils/validation.ts

```diff
@@ -3,6 +3,8 @@ import { getI18n } from '../plugins/a.i18n'
 export const validateEmail = (v: string) =>
   /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([^<>()[\].,;:\s@"]+\.)+[^<>()[\].,;:\s@"]{2,})$/i.test(v)
 
+export const validatePhone = (v: string) => /^[0-9+]{6,}$/.test(v)
```

Comment:
this may cover some cases only

Reply:
second thought - can't we just use `validator/lib/isMobilePhone` since we already have this package in nc-gui?

Reply:
second thought - can't we just use `validator/lib/isMobilePhone` since we already have this package in nc-gui?

Reply:
ok